<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1JFO5LO" name="Test" tests="3" failures="0" timestamp="24 Oct 2019 09:26:12 GMT" time="46.28" errors="0">
  <testcase name="@BeforeMethod openApp" time="0.0" classname="script.ValidLogin_Test">
    <failure type="org.openqa.selenium.WebDriverException" message="Timed out waiting for driver server to start.
Build info: version: &amp;apos;4.0.0-alpha-3&amp;apos;, revision: &amp;apos;8c567de6dc&amp;apos;
System info: host: &amp;apos;DESKTOP-1JFO5LO&amp;apos;, ip: &amp;apos;192.168.43.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_211&amp;apos;
Driver info: driver.version: ChromeDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting for driver server to start.
Build info: version: '4.0.0-alpha-3', revision: '8c567de6dc'
System info: host: 'DESKTOP-1JFO5LO', ip: '192.168.43.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_211'
Driver info: driver.version: ChromeDriver
	at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:203)
	at org.openqa.selenium.remote.service.DriverService.start(DriverService.java:189)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:210)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)
	at org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:75)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:163)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:150)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:105)
	at generic.BaseTest.openApp(BaseTest.java:26)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:5245/status] to be available after 20003 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:86)
	at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:198)
	... 40 more
Caused by: java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(FutureTask.java:205)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:84)
	... 41 more
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod openApp -->
  <testcase name="@AfterMethod closeApp" time="0.0" classname="script.ValidLogin_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at generic.BaseTest.closeApp(BaseTest.java:45)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod closeApp -->
  <testcase name="@BeforeMethod openApp" time="0.0" classname="script.VerifyProductVersion_Test">
    <skipped/>
  </testcase> <!-- @BeforeMethod openApp -->
  <testcase name="@AfterMethod closeApp" time="0.0" classname="script.VerifyProductVersion_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at generic.BaseTest.closeApp(BaseTest.java:45)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod closeApp -->
  <testcase name="@BeforeMethod openApp" time="0.0" classname="script.InvalidLogin_Test">
    <skipped/>
  </testcase> <!-- @BeforeMethod openApp -->
  <testcase name="@AfterMethod closeApp" time="0.0" classname="script.InvalidLogin_Test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at generic.BaseTest.closeApp(BaseTest.java:45)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod closeApp -->
  <testcase name="testValidLogin" time="0.002" classname="script.ValidLogin_Test">
    <skipped/>
  </testcase> <!-- testValidLogin -->
  <testcase name="testProductVerson" time="0.0" classname="script.VerifyProductVersion_Test">
    <skipped/>
  </testcase> <!-- testProductVerson -->
  <testcase name="testInvalidLogin" time="0.002" classname="script.InvalidLogin_Test">
    <skipped/>
  </testcase> <!-- testInvalidLogin -->
</testsuite> <!-- Test -->
